### PR TITLE
Typescript support

### PR DESCRIPTION
**Pull request by RobertSasak**
Thank you for your amazing work. I hope this types will make this library even better.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+declare module 'coordinate-parser' {
+    class CoordinateParser {
```
Comment (otto-dev) [inline index.d.ts:2]: This module does not provide any class `CoordinateParser`. Is this line correct?

File: index.d.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+        constructor(coordinateString: string)
+        getLatitude(): number
+        getLongitude(): number
+        isValidPosition(): boolean
```
Comment (otto-dev) [inline index.d.ts:6]: There is no method with this name in the source code.

**Review by otto-dev — COMMENTED** [review]
Thanks a lot, appreciate the contribution. Please clarify the above comments.

File: index.d.ts (hunk 2 above)
Comment (RobertSasak) [reply index.d.ts]: You are right. Removed.

File: index.d.ts (hunk 1 above)
Comment (RobertSasak) [reply index.d.ts]: Compiled JS code corresponds to a typescript class. I change name to Coordinates to match JS code. Thank you pointing this out.

Comment (otto-dev) [reply index.d.ts]: Thanks, I wasn't sure if the name here matters to TypeScript.

**Review by otto-dev — APPROVED** [review]
Thank you for this contribution, and for addressing the comments.

**Comment by jgeiger-cr** [comment]
Thanks for the great utility.  FYI the update above seems to break the Angular use case.  I've pinned the 1.0.3 release in the mean time.

`error TS2351: This expression is not constructable.
Type 'typeof import("coordinate-parser")' has no construct signatures.
38       const candidateAsLocation: any = new Coordinates(candidate);
`

**Comment by RobertSasak** [comment]
Can you please post a full code?

**Comment by otto-dev** [comment]
Ah dang. This shouldn't be a breaking change. Will look into it.

**Pull request merged**